FreeRTOS/FreeRTOS
# Updated Demo changes

Updated Demo changes

Description
-----------
add sin_addr.xIPV4 in xEchoAddr
Test Steps
-----------
Normal Demo steps

Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
